sympy/physics/units/quantities.py

```diff
@@ -7,7 +7,7 @@
 from __future__ import division
 
 from sympy.core.compatibility import string_types
-from sympy import sympify, Expr, Mul, Pow, S, Symbol, Add, AtomicExpr
+from sympy import sympify, Mul, Pow, S, Symbol, Add, AtomicExpr
```

Comment:
If you are making such change, can you also please make them in the alphabetical order? i.e `Add, AtomicExpr, Mul, Pow, S, Symbol, sympify". Then probably a squash of commits into a single one.